project_stock/views/project_task_view.xml

```diff
@@ -97,12 +103,14 @@
                 />
             </field>
             <xpath expr="///page[@name='extra_info']" position="before">
+                <!-- Field without groups used as domain in stock_analytic_* fields !-->
+                <field name="use_stock_moves" invisible="1" />
```

Comment:
No need to move this field.

Reply:
Yes it is necessary (the comment tries to clarify this).

The `stock_analytic_*` fields have the analytic groups and use the `use_stock_moves` field in the domain, so it should not have groups.

Reply:
OK, I understand, as you are placing in the page a group now.